modules/qtwidgets/rawdatareaderdialogqt.cpp

```diff
@@ -148,22 +225,27 @@ RawDataReaderDialogQt::RawDataReaderDialogQt() {
     mainLayout->addWidget(readOptionsBox, 5, 0, 3, 2);
     mainLayout->addWidget(buttonBox,      8, 1);
     setLayout(mainLayout);
+	// Set default to USHORT. Will Update default values of ranges 
+	bitDepth_->setCurrentIndex(2);
```

Comment:
Why default to USHORT? I guess most of you data is USHORT, most of mine isn't. So why ushort :)

Reply:
Better than uchar :) 
We have ok initial settings for USHORT. Others are a bit cumbersome since we don't have any good guess on initial data mapper values.